cla_public/static-src/stylesheets/main.scss

```diff
@@ -389,3 +389,31 @@ dl.govuk-list.govuk-list--bullet {
 .laa-flee-button-container {
   display:none;
 }
+
+.cla-staging {
+  .govuk-header__container {
+    border-bottom-color:govuk-colour("pink");
+  }
+  .govuk-tag {
+    background-color:govuk-colour("pink");
+  }
+}
+
+.cla-production {
+  .govuk-header__container {
+    border-bottom-color:govuk-colour("red");
+  }
+  .govuk-tag {
+    background-color:govuk-colour("red");
+  }
+}
+
+.cla-dev {
+  .govuk-header__container {
+    border-bottom-color:govuk-colour("turquoise");
+  }
+  .govuk-tag {
```

Comment:
On reflection, I think we should make this more specific:
Instead of using `.govuk-tag`, use `.govuk-phase-banner__content__tag`